### PR TITLE
HG-80: List

### PR DESCRIPTION
**Pull request by dtomaszewski88**

### CODE REVIEW

File: src/components/List/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@further-ventures/fv-react-ui-components.chip",
```
Comment (Nagibaba) [inline src/components/List/package.json:2]: list

Comment (AlexP-anec) [reply src/components/List/package.json:2]: done!

File: src/components/List/ListItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+  return (
+    <li
+      {...rest}
+      className={classNames(className, 'relative items-center bg-background transition-all ease-out duration-300', {
+        ['text-text-disabled']: disabled,
+        ['hover:bg-default-extra-light active:bg-default-light']: !disabled,
+      })}
+    >
+      <button onClick={control !== 'checkbox' ? handleSelect : () => {}} className='flex flex-nowrap w-full text-left'>
```
Comment (SergeySychevskiy) [inline src/components/List/ListItem/index.tsx:74]: Why do we need the html button component here? By default it has css styles, which we probably have to reset. But I don't really see the point to have button, do we really need it?

Comment (AlexP-anec) [reply src/components/List/ListItem/index.tsx:74]: We have onClick attribute and usually lint rules will scream about it. But I can also change into a span or smth

**Review by SergeySychevskiy — APPROVED** [review]

File: src/components/List/ListItem/index.tsx (hunk 2 above)
Comment (SergeySychevskiy) [reply src/components/List/ListItem/index.tsx:74]: Since there are no default button styles issue it's fine :)

**Pull request merged**